bundles/org.openhab.binding.emotiva/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -489,7 +489,7 @@
 
 	<channel-type id="selected-movie-music">
 		<item-type>String</item-type>
-		<label>User Selected Mode</label>
+		<label>User Selected Movie or Music Mode</label>
```

Comment:
Labels should be 2-3 words and < 23 chars. 
‘Media Mode’ or something similar?

Reply:
Media Mode is a good suggestion.